numba/unicode.py

```diff
@@ -31,6 +31,9 @@
 from numba.targets.hashing import _Py_hash_t
 from numba.unsafe.bytes import memcpy_region
 
+import llvmlite.binding as ll
+import string_conversion_ext
```

Comment:
`string_conversion_ext` should not be a top-level module.  See comment in setup.py